kvdb/common_sql/readwrite_bucket.go

```diff
@@ -90,6 +90,15 @@ func (b *readWriteBucket) Get(key []byte) []byte {
 		panic(err)
 	}
 
+	// When an empty byte array is stored as the value, Sqlite will decode
```

Comment:
We ran into this on the Taro side, but then upgraded either `sqlc` or `sqlite` and it was resolved, so potentially something to check up on.